spec/features/help/site_tour_spec.rb

```diff
@@ -1,19 +1,10 @@
 require 'rails_helper'
 
 describe 'Site tour' do
-  context 'When loading the initial search page while logged in', pending_updates: true do
+  context 'When loading the initial search page while logged in' do
     before :each do
       Capybara.reset_sessions!
-      login
-      load_page '/search'
-      wait_for_xhr
-      dismiss_banner
-      find_link('Manage user account').click
-      find_link('Show Tour').click
-      # The tour loads with 'Do not Show Again' unchecked - closing it immediately sets the preference to false
-      # for this test.  This ensures the right starting point.
-      find_button('End Tour').click
-      page.execute_script 'window.location.reload()'
+      load_page :root, authenticate: 'edsc', keep_tour_open: true
       wait_for_xhr
```

Comment:
Shouldn't need this `wait_for_xhr` as `load_page` calls that at the end of the method.